### PR TITLE
Iconbar

### PR DESCRIPTION
**Pull request by serapath**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,63 +1,73 @@
...
 # Remix
-
 Remix (aka. Browser-Solidity) is a browser-based Solidity compiler and IDE.
 
 Visit [https://remix.ethereum.org](https://remix.ethereum.org) to use;
 it will always deliver the latest version.
 
+
 ## Offline Usage
+The `gh-pages` branch always has the latest stable build of Browser-Solidity.
```
Comment (chriseth) [inline README.md]: Please keep the name "Remix".

File: sc.js [hunk 2]
```diff
@@ -0,0 +1,81 @@
+var sauceConnectLauncher = require('sauce-connect-launcher')
```
Comment (chriseth) [inline sc.js]: I would say the name of the file is too short and thus not descriptive enough for the root directory.

File: README.md (hunk 1 above)
Comment (serapath) [inline README.md]: Oh sure - no problem - that was an unintentional change - not even aware i changed it, but i guess i did.

File: sc.js (hunk 2 above)
Comment (serapath) [inline sc.js]: ok, so i'll put it into `ci/` ?

**Comment by serapath** [comment]
sorry, there were unrelated changes in this pull request - fixed now and all checks pass

File: index.html [hunk 3]
```diff
@@ -39,8 +39,8 @@
 	<body>
 
 		<div id="editor">
-			<span class="newFile" title="New File"><i class="fa fa-file-code-o"></i></span>
-			<span class="uploadFile" title="Open local file"><label class="fa fa-folder-open-o"><input type="file" class="inputFile" multiple /></label></span>
+			<span class="newFile" title="New File"><i class="fa fa-plus-circle" aria-hidden="true"></i></span>
```
Comment (axic) [inline index.html]: What is the `aria-hidden="true"` for?

**Pull request merged**